### PR TITLE
.NET Monitor release

### PR DESCRIPTION
**Pull request by mthalman**
Related to https://github.com/dotnet/dotnet-docker/issues/2852

### CODE REVIEW

File: src/monitor/5.0/alpine/amd64/Dockerfile [hunk 1]
```diff
@@ -26,6 +26,9 @@ ENV \
     ASPNETCORE_URLS= \
     # Disable debugger and profiler diagnostics to avoid diagnosing self.
     COMPlus_EnableDiagnostics=0 \
+    # Default Filter
```
Comment (MichaelSimons) [inline src/monitor/5.0/alpine/amd64/Dockerfile:29]: The VerifyEnvironmentVariables test should be updated to cover these - https://github.com/dotnet/dotnet-docker/blob/main/tests/Microsoft.DotNet.Docker.Tests/MonitorImageTests.cs#L85.  

This shouldn't block the release but should be addressed before RTM.

**Review by MichaelSimons — APPROVED** [review]

File: src/monitor/5.0/alpine/amd64/Dockerfile (hunk 1 above)
Comment (mthalman) [reply src/monitor/5.0/alpine/amd64/Dockerfile:29]: Logged https://github.com/dotnet/dotnet-docker/issues/2854

**Review by jander-msft — APPROVED** [review]

**Review by jander-msft — CHANGES_REQUESTED** [review]
@mthalman, we're requesting to cancel the release at this time to iron out an issue that we discovered. I'll sync up with you again in the near future when we're ready to release.

**Comment by mthalman** [comment]
I'll keep this open for now and just update with the new changes.

**Comment by jander-msft** [comment]
@mthalman, please merge the new dotnet-monitor version (5.0.0-preview.5.21302.5) from the nightly branch.

**Comment by mthalman** [comment]
> @mthalman, please pull in the new dotnet-monitor version (5.0.0-preview.5.21302.5) from the nightly branch.

Thanks for the reminder.  Updated.

**Review by jander-msft — APPROVED** [review]

**Pull request merged**